reimbursement.md

```diff
@@ -394,6 +398,12 @@ fileName:transaction.json
 
 The reimbursement request and status update can fail for various reasons. Your application should handle these errors accordingly.
 
+<h3 id="retry">Retry</h3>
```

Comment:
You are doing with an h3 instead of ### cause you need the `id=retry`?

Reply:
Yes, I need it so that I can link to it via anchor tag (reimbursement#retry). Only H1 and 2 have them by default, but not H3 headings.